### ISSUE TITLE
Enable Custom Elements within the <template> tag

### ISSUE THREAD

**Issue by ciampo** [PolymerElements/iron-list#235]
At the moment, it's not possible to include any Custom Element (i.e. Shadow DOM nodes) inside an iron-list <template>. User gets the warning

 `iron-list requires a template to be provided in light-dom`

Given the nature of Web Components, I find this feature quite essential to the `iron-list` component

Marco

**Comment by blasten** [comment]
Yes, this isn't supported yet. I think this feature needs discussion, but afaik you will still need to provide a Template in the light DOM. e.g.
```html
<iron-list>
  <template>
    <content></content>
  </template> 
</iron-list>
```

**Comment by ronnyroeller** [comment]
+1

**Comment by timove** [comment]
+1, see #139

**Comment by blasten** [comment]
Fixed in [v1.4.0](https://github.com/PolymerElements/iron-list/releases/tag/v1.4.0)
